upload/bitpay.php

```diff
@@ -676,6 +708,7 @@ function plgVmConfirmedOrder($cart, $order)
         if (isset($response['url']))
         {
             header('Location: ' . $response['url']);
+			$cart->emptyCart ();

```

Comment:
This line isn't going to be called since the header() function forwards the execution to the external page.